### ISSUE TITLE
`import io` fails on Linux

### ISSUE THREAD

**Issue by dsandstrom** [irohiroki/RubyBlockConverter#3]
Upon installing your plug-in I receive this in the console: 

```
Traceback (most recent call last):
  File "./sublime_plugin.py", line 62, in reload_plugin
  File "./RubyBlockConverter.py", line 1, in <module>
    import io
  File ".\io.py", line 63, in <module>
ImportError: No module named _fileio
```

I using Linux Mint 13 x64 and Sublime Text 2 Build 2220.  Apparently, _fileio doesn't  exist in ST's python in Linux.  I found similar issues on the other Sublime packages: https://github.com/timonwong/OmniMarkupPreviewer/issues/8 and https://github.com/vi4m/sublime_python_imports/issues/3.  And a relevant [SO question](http://stackoverflow.com/questions/15180537/how-to-include-third-party-python-packages-in-sublime-text-2-plugins).

**Comment by irohiroki** [comment]
Thank you for the info.
can you try `linux` branch of this repo?
if it's ok I'll merge it to master.

**Comment by dsandstrom** [comment]
Unfortunately not, the script loads, but after trying the key bindings on each block type, I get:

```
Traceback (most recent call last):
  File "./sublime_plugin.py", line 362, in run_
  File "./RubyBlockConverter.py", line 124, in run
  File "./RubyBlockConverter.py", line 39, in match_blocks
NameError: global name 'io' is not defined
Traceback (most recent call last):
  File "./sublime_plugin.py", line 362, in run_
  File "./RubyBlockConverter.py", line 73, in run
  File "./RubyBlockConverter.py", line 39, in match_blocks
NameError: global name 'io' is not defined
```

**Comment by irohiroki** [comment]
fixed with #4.
